styles/crunchyroll/catppuccin.user.css

```diff
@@ -881,9 +881,17 @@
       }
     }
     // CRUNCHYLISTS PAGE - END
+    // appies outside of .erc-account as well.
```

Comment:
Where exactly? I did not find any cases.

Reply:
hmm im not sure, i think on another device it had been there i seem to be wrong tho.